### PR TITLE
REM-1372: Fix list_blocks with unexist parameters

### PR DESCRIPTION
**Pull request by Alladin9393**
### Jira references
- Ticket (task) — https://remmeio.atlassian.net/browse/REM-1372

### CODE REVIEW

File: testing/unit/rpc/test_block_info.py [hunk 1]
```diff
@@ -596,6 +613,23 @@
...
 
+@pytest.mark.asyncio
+@pytest.mark.parametrize('non_existing_head', ['a'*128])
+async def test_list_blocks_with_non_existing_head(request_, non_existing_head):
+    """
+    Case: list blocks with non existing head.
+    Expect: list of blocks not found error message.
+    """
+    request_.params = {
+        'start': non_existing_head,
```
Comment (anastasiia-bilova) [inline testing/unit/rpc/test_block_info.py]:
```suggestion
        'head': non_existing_head,
```

File: testing/unit/rpc/test_block_info.py [hunk 2]
```diff
@@ -389,6 +389,23 @@
     assert 'Header signature is not of a blockchain token type.' == error.value.message
 
 
+@pytest.mark.asyncio
+@pytest.mark.parametrize('non_existing_start', ['a'*128])
+async def test_list_blocks_with_non_existing_start(request_, non_existing_start):
```
Comment (anastasiia-bilova) [inline testing/unit/rpc/test_block_info.py]: @Alladin9393, add test for parameter ids.

File: testing/unit/rpc/test_block_info.py (hunk 1 above)
Comment (Alladin9393) [reply testing/unit/rpc/test_block_info.py]: @anastasiia-bilova, done.

File: testing/unit/rpc/test_block_info.py (hunk 2 above)
Comment (Alladin9393) [reply testing/unit/rpc/test_block_info.py]: @anastasiia-bilova, done.

**Pull request merged**